tuning/sft_trainer.py

```diff
@@ -477,12 +479,43 @@ def parse_arguments(parser, json_config=None):
     )
 
 
-def main(**kwargs):  # pylint: disable=unused-argument
-    logger = logging.get_logger("__main__")
+def set_log_level(parsed_training_args, logger_name=__name__):
```

Comment:
Just a suggestion: imo it will look better if this were to be moved to a separate file `utils/logging.py`

Reply:
Also since `parsed_training_args` looks similar to `train_args` it might be simple to rename this argument.

Reply:
Sure!